src/Application/Actions/Donations/Create.php

```diff
@@ -147,8 +150,20 @@ protected function action(Request $request, Response $response, array $args): Re
                 $donation->setCampaign($campaign);
             }
 
+            if ($forNewPaymentElement) {
+                // We don't need to set specific payment methods here, they are set on the client side by the payment
+                // element.
+                $stripeMethodProperties = [
+                    'automatic_payment_methods' => [
+                    'enabled' => true,
+                    'allow_redirects' => 'always',
```

Comment:
@bdsl Does `never` cause any problems here, if our account has no redirect based methods switched on?

Reply:
I'd have to test it to know for sure. But the frontend is configured in the open PR to only do redirects if required, so I don't think there's any harm I can see from allowing them. I've set it to redirect to the thank you page. Is there any advantage to never? Even if does work now it would mean any payment method we try to add that requires a redirect would get filtered out. https://stripe.com/docs/api/errors#errors-setup_intent-automatic_payment_methods-allow_redirects

Reply:
See https://github.com/thebiggive/matchbot/pull/588